README.md

```diff
@@ -40,7 +48,9 @@ var auth0 = new AuthenticationClient({
 });
 ```
 
-## Management API Client
+#### Management API Client
+
+> Note: When using at browser you should use `telemetry: false`.
```

Comment:
I know this has only been moved, but I am not sure what this is supposed to say ...

Reply:
Yeah I reworded this, then removed it, then readded after chatting with Adam. Will fix up the wording